app/src/services/google-analytics-service.js

```diff
@@ -9,13 +9,13 @@ angular.module('otServices')
      */
     .factory('otGoogleAnalytics', ['$location', function ($location) {
         function trackPageView () {
-            if ($location.host() === 'www.targetvalidation.org') {
+            if (typeof ga !== 'undefined') {
```

Comment:
We think we need to keep the locaiton name check somewhere, though maybe not here. This is to avoid poisoning the analytics from other sites (e.g. QA, localhost, etc) particularly for the events.

Reply:
Since the default behavior (in this PR) is to _not_ have any analytics, most of your deployments can remain unchanged. The only deployment for which you would have to make a change is the one for the production server where you would like to have analytics.